packages/scss/src/components/_actionIcon.scss

```diff
@@ -0,0 +1,20 @@
+.actionIcon {
+	background: transparent;
+	border: 0;
+	border-radius: 50%;
+	color: _color("text.default");
+	cursor: pointer;
+	height: 2rem;
```

Comment:
I do think these buttons are meant to be linked to its parent size. Size options should exist to make them larger or smaller relative to their parent size.